### DISCUSSION TITLE
What's the difference between Windows 10 and Windows 10 (ExplorerPatcher)?

### DISCUSSION THREAD

**Discussion by GuestTheGamer** [valinet/ExplorerPatcher#3654] [Q&A]
![Screenshot 2024-09-09 175902](https://github.com/user-attachments/assets/525209a1-9f63-4254-b184-fbae342ebf9d)

**Comment by pyrates999** [comment]
please see this: https://github.com/valinet/ExplorerPatcher/wiki/ExplorerPatcher's-taskbar-implementation

**Comment by Amrsatrio** [comment]
Simple answer: "Windows 10" taskbar style is gone in 24H2. Users who want to continue to have a 10 taskbar experience should use "Windows 10 (ExplorerPatcher)" taskbar style. It is available on 22H2 so that users can try it and report serious bugs early before it becomes the only option.
